android/app/src/main/java/com/microsoft/reactnativecodepush/CodePush.java

```diff
@@ -0,0 +1,412 @@
+package com.microsoft.reactnativecodepush;
```

Comment:
In case we have a native code push SDK, maybe its better to keep the react native prefix there.